src/main/java/mate/academy/lib/Injector.java

```diff
@@ -47,8 +48,8 @@ private Class<?> findClassExtendingInterface(Class<?> certainInterface) {
             }
         }
         throw new RuntimeException("Can't find class which implements "
-                + certainInterface.getName()
-                + " interface and has valid annotation (Dao or Service)");
+            + certainInterface.getName()
+            + " interface and has valid annotation (Dao or Service)");
```

Comment:
The error message mentions 'Service' annotation, but the implementation only checks for the 'Dao' annotation. Ensure consistency between the error message and the actual logic.

Reply:
It isn't my implementation